build.gradle

```diff
@@ -20,10 +20,12 @@ repositories {
 
 dependencies {
     compile name: 'touch-image-view'
+    compile 'com.android.support:support-v4:22.2.1'
     compile project(':owncloud-android-library')
     compile 'com.android.support:support-v4:22.2.1'
     compile 'com.jakewharton:disklrucache:2.0.2'
     compile 'com.android.support:appcompat-v7:22.2.1'
+    compile 'com.getbase:floatingactionbutton:1.10.1'
```

Comment:
The problem with the design support lib is that it does not support the speed dial but just simple FABs. That is why I added the third party lib. Else we would have to write our own implementation.